Project.toml

```diff
@@ -31,12 +31,20 @@ JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 Legolas = "741b9549-f6ed-4911-9fbf-4a1c0c97f0cd"
 LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+LineSearches = "d3d80556-e9d4-5f37-9878-2ab0fcc64255"
+LinearSolve = "7ed4a6bd-45f5-4d41-b270-4a48e9bafcae"
```

Comment:
These were already core deps, but for interactive use it's nice to have all core deps in the root environment as well.